CMakeLists.txt

```diff
@@ -204,13 +196,6 @@ if(NOT WITH_STL STREQUAL "OFF")
   endif()
 endif()
 
-if(DEFINED WITH_OTLP)
```

Comment:
As this triggers fatal error, should fine to keep it?

Reply:
It is also safe to assume `WITH_OTLP` is no longer in use, because it would cause an error.

I prefer to finally remove old options that no longer exist, to simplify maintenance.

If we do not remove it now, when then ?